docs/reference/method/MongoDBClient__construct.txt

```diff
@@ -47,6 +57,48 @@ demonstrates how to connect to a replica set with a custom read preference:
        ]
    );
 
+Connecting with SSL and Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example demonstrates how to connect to a MongoDB replica set with
```

Comment:
Might be good to mention here that the URI is a standard MongoDB connection string and link to those docs.